frontend/test/metabase/scenarios/question/notebook.cy.spec.js

```diff
@@ -97,6 +98,25 @@ describe("scenarios > question > notebook", () => {
       cy.contains("Showing 1 row");
     });
 
+    it.skip("should show correct column title with foreign keys (Issue #11452)", () => {
+      // Join tables with foreign keys
+      openOrdersTable();
+      cy.get(".Icon-notebook").click();
+      cy.findByText("Join data").click();
+      cy.findByText("Reviews").click();
+      cy.findByText("Product ID").click();
+      popover().within(() => {
+        cy.findByText("Product ID").click();
+      });
+
+      // Column titles should not include 'Product'
+      cy.get(".QueryBuilder .Icon-sum").click();
+      cy.findByText("Pick a column to group by").click();
+      cy.get(".ReactVirtualized__List").within(() => {
+        cy.findAllByText("Product").should("not.exist");
```

Comment:
I think we _should_ find "Product" in the list. My understanding is that the first appearance of "Product" below should be "Review", but the second is correct.
![image](https://user-images.githubusercontent.com/691495/89680404-df4e6a00-d8c0-11ea-9461-b8147dafeb76.png)

Reply:
Gotcha, I think I misunderstood; I'll correct that.